packages/arb-token-bridge-ui/src/components/common/SafeImage.tsx

```diff
@@ -11,20 +11,25 @@ export function SafeImage(props: SafeImageProps) {
   const [validImageSrc, setValidImageSrc] = useState<false | string>(false)
 
   useEffect(() => {
+    let mounted = true
```

Comment:
Need to track this, otherwise there are side effects when changing networks and token logos not updating